app/src/main/java/org/oppia/android/app/player/exploration/ExplorationFragmentPresenter.kt

```diff
@@ -4,30 +4,49 @@ import android.content.Context
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.ImageButton
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.forEach
 import androidx.fragment.app.Fragment
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Transformations
+import javax.inject.Inject
+import kotlinx.android.synthetic.main.exploration_activity.*
+import kotlinx.android.synthetic.main.exploration_activity.view.*
```

Comment:
These are deprecated--we shouldn't use them. For Oppia, prefer using databinding or regular findViewById to access views.

Reply:
done